src/components/CodeSnippet/CodeSnippet.test.tsx

```diff
@@ -1,40 +1,41 @@
-import { mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import React from "react";
 
 import CodeSnippet, { CodeSnippetBlockAppearance } from "./index";
 
 describe("CodeSnippet ", () => {
   it("renders a code block", () => {
-    const wrapper = mount(<CodeSnippet blocks={[{ code: "Test" }]} />);
-    expect(wrapper.find(".p-code-snippet__block").text()).toEqual("Test");
+    render(<CodeSnippet blocks={[{ code: "Test" }]} />);
+    expect(screen.getByText("Test")).toBeInTheDocument();
   });
 
   it("renders title for a code block", () => {
-    const wrapper = mount(
-      <CodeSnippet blocks={[{ title: "Title", code: "Test" }]} />
+    render(<CodeSnippet blocks={[{ title: "Title", code: "Test" }]} />);
+    expect(screen.getByRole("heading", { name: "Title" })).toHaveClass(
```

Comment:
That's a good example of how I think we could write tests with RTL - use semantic RTL selectors, and check class names on them if needed.